rfc-164-retire-email-alert-monitoring-as-a-separate-app.md

```diff
@@ -0,0 +1,76 @@
+
+As a side-effect of adding this, we could also potentially query every alert email in the given period to determine how many of our alerts on average are getting delivered (and perhaps identify recipients who no longer exist).
+
+This isn't quite as end-to-end as the current system (if something was delivered, does that mean it's actually in a given person's inbox?), but since any record that Notify deems delivered is at the absolute boundaries of anything actionable by GOV.UK, there seems to be little to gain by tracking the alerts further than that.
+
+[query notify as to their current status]: https://docs.notifications.service.gov.uk/ruby.html#get-the-status-of-one-message
+
+## Technical Details
+
+### How email-alert-monitoring currently works for Medical Alerts
+
+The app is called as a cronjob once every 30 minutes. It opens the [RSS feed] for medical alert updates, grabs the first 5, then filters out of those five any that were published in the last hour. For the remaining items, if any, it makes a list of the subject lines of the emails that should have gone out. It then uses the GMail API to read the inbox of govuk_email_check@digital.cabinet-office.gov.uk to check for the existence of those emails. If any are missing, an alert happens. Minimum time between publishing and alert is 1 hour.
+
+[RSS feed]: https://www.gov.uk/drug-device-alerts.atom
+
+### How email-alert-monitoring currently works for Travel Advice Alerts
+
+The app is called as a cronjob once every 15 mintues. It opens a [healthcheck URL] on the travel-advice-publisher app, which contains a list of all travel alerts published in the last 2 days, but excluding the last 150 minutes. For these, it makes a list of the subject lines of the emails that should have been sent out, as well as the time (this is because more than one travel advice alert migt be issued for the same country in a two day period, so matching purely on the subject line isn't sufficient). It then uses the GMail API to read the inbox of govuk_email_check@digital.cabinet-office.gov.uk to check for the existence of those emails. If any are missing, an alert happens. Minimum time between publishing and alert is 2 and a half hours.
+
+[healthcheck URL]: https://travel-advice-publisher.publishing.service.gov.uk/healthcheck/recently-published-editions
+
+### Problems with matching subject lines
+
+Problems can occur when trying to match by subject line if the title of the alert is altered after the email has been sent out. The [RSS feed] or [healthcheck URL] will contain a title that differs from the subject line of the email sent out. This will cause a failed match, and an alert will go off, even though for practical purposes the email has been sent. This causes false alarms, and needs someone to add a [hard-coded exception] into the matching code, increasing toil.
+
+[hard-coded execption]: https://github.com/alphagov/email-alert-monitoring/blob/main/lib/email_verifier.rb#L8-L36
+
```

Comment:
Might be worth adding a "Problems with matching timestamps" section too, since they add another vector for false positives into the mix - see https://github.com/alphagov/email-alert-monitoring/pull/106